kratos/tests/cpp_tests/geometries/test_quadrature_point_geometry.cpp

```diff
@@ -104,5 +131,44 @@ namespace Kratos {
             KRATOS_CHECK_EQUAL(p_this_quadrature_point->ShapeFunctionsValues().size2(), 3);
         }
 
+        KRATOS_TEST_CASE_IN_SUITE(QuadraturePointGeometry2dCopyConstructor, KratosCoreFastSuite)
```

Comment:
I think it would be good to additionally check that if the original geometry is modified (`p_this_quadrature_point`) that the copied is not affected (`geom`)

I am not sure if this is possible since I don't exactly know the interface
Can you check please?

Reply:
It is actually hard to do that with the current capabilities...

However, I can do that maybe as soon as the container is there.